WeakAurasOptions/OptionsFrames/Update.lua

```diff
@@ -1641,7 +1642,7 @@ local methods = {
     self.closeButton:SetEnabled(true)
     OptionsPrivate.Private.callbacks:Fire("Import")
 
-    self:Close()
+    self:Close(pendingPickData.id)
```

Comment:
You are passing in the id here, which I think is a good idea.

But then the Close function's argument should not be called "wasSuccesful", but id